lizard_efcis/models.py

```diff
@@ -118,6 +125,21 @@ class Locatie(models.Model):
 
     objects = models.GeoManager()
 
+    def save(self, *args, **kwargs):
+        if self.x1 and self.y1:
```

Comment:
Officieel mag x1 0 zijn, dus zou "self.x1 is not None" beter zijn, maar omdat x2 tussen 300000 en 600000 ligt... Laat maar zo :-)